pages/cloud/storage/nas_nfs/guide.en-gb.md

```diff
@@ -1,236 +1,194 @@
 ---
-title: 'Mount your NAS via NFS share'
+title: Mounting HA-NAS via NFS share
 slug: nas-nfs
-excerpt: 'Find out how to mount a NAS over an NFS share here.'
+excerpt: Find out how to connect to your HA-NAS using an NFS share
 section: NAS
 order: 03
 ---
 
-**Last updated 21st February 2022**
-
-## Requirements
-
-This guide allows you to mount NFS on the most common distributions. To mount an NFS share, you must have:
-
-- a [Dedicated Server](https://www.ovhcloud.com/en-gb/bare-metal/) **or** a [VPS](https://www.ovhcloud.com/en-gb/vps/) **or** a [Public Cloud Instance](https://www.ovhcloud.com/en-gb/public-cloud/).
-- a [HA-NAS solution](https://www.ovh.co.uk/nas/).
-- an NFS-compatible distribution.
+**Last updated 16th September 2022**
 
+## Objective
 
-### Linux
+The OVHcloud HA-NAS service allows you to manage file storage that can be accessed over a network.
 
-Compatibility: Debian  & Ubuntu
+**This guide explains how to access your HA-NAS via NFS on the most common distributions.**
 
-To mount an NFS share on Linux, you must:
+> [!warning]
+>OVHcloud is providing you with services for which you are responsible, with regard to their configuration and management. It is therefore your responsibility to ensure that they they function correctly.
+>
+>This guide is designed to assist you in common tasks as much as possible. Nevertheless, we recommend that you contact a [specialist service provider](https://partner.ovhcloud.com/en-gb/directory/) or reaching out to [our community](https://community.ovh.com/en/) if you face difficulties or doubts concerning the administration, usage or implementation of services on a server.
+>
 
-- Connect to the server in SSH.
-- Install the "nfs-client" package via command:
+## Requirements
 
+- An [OVHcloud HA-NAS solution](https://www.ovhcloud.com/en-gb/storage-solutions/nas-ha/)
```

Comment:
In my opinion, translating the product name is an issue , but I can understand this is a commercial decision.
However if the name is translated or not, we must take this in account in the URL path

Reply:
Yes but why?

And why the path is not?

Reply:
Not our call inside the documentation team. Please ask website/marketing teams.

Reply:
Notice that we had some inconsistencies if we look in the whole doc:
```bash
❯ grep -r "NAS-HA" pages/ | grep "en.gb"
pages/platform/kubernetes-k8s/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.en-gb.md:title: Configuring multi-attach persistent volumes with OVHcloud NAS-HA
pages/platform/kubernetes-k8s/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.en-gb.md:excerpt: 'Find out how to configure a multi-attach persistent volume using OVHcloud NAS-HA'
pages/platform/kubernetes-k8s/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.en-gb.md:OVHcloud Managed Kubernetes natively integrates Block Storage as persistent volumes. This technology may however not be suited to some legacy or non cloud-native applications, often requiring to share this persistent data accross different pods on multiple worker nodes (ReadWriteMany or RWX). If you would need to do this for some of your workloads, one solution is to use NFS volumes. [OVHcloud NAS-HA](https://www.ovh.co.uk/nas/) is a managed solution that lets you easily configure an NFS server and multiple NFS volumes. In this tutorial we are going to see how to configure your OVHcloud Managed Kubernetes cluster to use [OVHcloud NAS-HA](https://www.ovh.co.uk/nas/) as an NFS provider for [Kubernetes Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/).
pages/platform/kubernetes-k8s/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.en-gb.md:It also assumes you have an OVHcloud NAS-HA already available. If you don't, you can [order one in the OVHcloud Control Panel](https://www.ovh.com/manager/dedicated/#/configuration/nas).
pages/platform/kubernetes-k8s/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.en-gb.md:Your NAS-HA can expose multiple partitions, and supports a variety of protocols. Each partition is accessible only from a specific range of IPs. We will create one exposing NFS and make it accessible from your Kubernetes worker nodes.
pages/platform/kubernetes-k8s/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.en-gb.md:Access the UI for OVHcloud NAS-HA by clicking the *NAS and CDN* menu in the [Server section of the OVHcloud Control Panel](https://www.ovh.com/manager/dedicated)
pages/platform/kubernetes-k8s/configuring-multi-attach-persistent-volumes-with-ovh-nas-ha/guide.en-gb.md:Congratulations, you have successfully set up a multi-attach persistent volume with OVHcloud NAS-HA!
pages/platform/kubernetes-k8s/known-limits/guide.en-gb.md:You can manually [configure multi-attach persistent volumes with NAS-HA](../Configuring-multi-attach-persistent-volumes-with-ovhcloud-nas-ha/).
pages/cloud/storage/nas_cifs/guide.en-gb.md:- CIFS mounting for a NAS-HA:
pages/account/billing/how_to_cancel_services/guide.en-gb.md:&emsp;&emsp;[NAS-HA Server](#nas)<br />
```

Reply:
Duly noted, we'll do the necessary fixes to align them all.

Reply:
> The product name is translated on https://www.ovhcloud.com/en-gb/storage-solutions/nas-ha/

But not in the URL